src/components/common/Options/ContextOptions.tsx

```diff
@@ -0,0 +1,34 @@
+import { ContextOptionsProps } from './ContextOptions.types';
+
+export default function ContextOptions({
+  options = [
+    { id: 1, name: '적용하기' },
+    { id: 2, name: '수정하기' },
+    { id: 3, name: '삭제하기' },
+  ],
+  onClick,
+}: ContextOptionsProps) {
```

Comment:
이렇게 표현되는게 의도를 드러내기에 더 좋을 것 같습니다~
```suggestion
const DEFAULT_OPTIONS = [
    { id: 1, name: '적용하기' },
    { id: 2, name: '수정하기' },
    { id: 3, name: '삭제하기' },
  ];

export default function ContextOptions({
  options = DEFAULT_OPTIONS,
  onClick,
}: ContextOptionsProps) {
```